plugins/org.jkiss.dbeaver.launcher/src/org/eclipse/equinox/launcher/Main.java

```diff
@@ -1841,28 +1843,31 @@ private String resolveLocation(String source, String var, String location) {
         return result.replaceFirst("^~", System.getProperty(PROP_USER_HOME));
     }
 
-    private String getProductVersion() {
-        String appVersion = "";
+    private String getProductProperties() throws IOException {
```

Comment:
It's not obvious that this method returns a path to product properties and not product properties themselves.

Also, add annotations, please.